### PR TITLE
[COC-65] Delete Account Screen

### PR DESCRIPTION
**Pull request by shape-shifters**
roundstyle.xml added for buttons shape and color

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,123 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (Alaple) [inline .idea/codeStyles/Project.xml]: Please delete this file

File: .idea/codeStyles/codeStyleConfig.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (Alaple) [inline .idea/codeStyles/codeStyleConfig.xml]: Please delete this file

**Review by Alaple — CHANGES_REQUESTED** [review]
Before merge please delete the files from the `.idea` folder

**Review by manuelfalonso — CHANGES_REQUESTED** [review]
Falta la logica del fragment

**Review by Alaple — APPROVED** [review]
image

**Review by manuelfalonso — APPROVED** [review]
lgtm

**Pull request merged**